d2renderers/d2svg/d2svg.go

```diff
@@ -493,6 +493,17 @@ func renderOval(tl *geo.Point, width, height float64, style string) string {
 	return fmt.Sprintf(`<ellipse class="shape" cx="%f" cy="%f" rx="%f" ry="%f" style="%s" />`, cx, cy, rx, ry, style)
 }
 
+func renderDoubleCircle(tl *geo.Point, width, height float64, style string) string {
+	rx := width / 2
+	ry := height / 2
+	cx := tl.X + rx
+	cy := tl.Y + ry
+	return fmt.Sprintf(`<ellipse class="shape" cx="%f" cy="%f" rx="%f" ry="%f" style="%s" />
```

Comment:
you can call `renderOval` twice here.
another alternative is to remove this function and call `renderOval` twice in the main one